dev.docker-compose.yml

```diff
@@ -10,3 +10,4 @@ services:
       XDG_CACHE_HOME: "/app/.cache"
     volumes:
       - ./:/app
+      - ${MY_SSH_DIR:-/tmp/__spiff_nossh}:${MY_SSH_DIR:-/tmp/__spiff_nossh}
```

Comment:
This is the first time the dot SSH directory has been mounted? Does debranch need it?

Reply:
By default it runs a fetch and prune, which requires network access - so if you did an ssh clone it needs it. You have to opt in to having the ssh directory mounted, so if you don't want to fetch and prune you can run the offline version, or if you did an https remote i'd imagine it would work as well.